minifi-assembly/pom.xml

```diff
@@ -261,7 +261,7 @@ limitations under the License.
         <!-- nifi.properties: web properties -->
         <nifi.web.war.directory>./lib</nifi.web.war.directory>
         <nifi.web.http.host />
-        <nifi.web.http.port>8080</nifi.web.http.port>
```

Comment:
I switched it because we're planning on getting rid of the UI anyway so currently it is primarily for debugging purposes and it's easier to debug when it doesn't have a conflicting default port as NiFi. If it makes more sense to switch it back I can then I would also change it in the ConfigTransformer (defaults to 8081 during transformation).